ts/util/logger/rotatingPinoDest.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022 Signal Messenger, LLC
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import fs from 'fs';
+import pino from 'pino';
+
+import { DURATION } from '../../session/constants';
+
+/**
+ * Keep at most rotated 3 files, so  4 files total including the "current" one
+ */
+const MAX_ROTATIONS = 3;
```

Comment:
I believe we discussed and wanted to keep 5 days of logs to cover logging over the weekend?

Reply:
that's 3 + 1, should be enough for the weekend? I am worried it is going to slow down the app drastically on export.